src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -315,23 +315,25 @@ private int parseCvrFile(
             }
             Integer candidateId = (Integer) rankingMap.get("CandidateId");
             String candidateCode = candidateId.toString();
-            Set<String> candidates = contestIdToCandidateCodes.get(contestId);
-            if (!candidates.contains(candidateCode)) {
+            String candidateName = config.getNameFor(candidateCode);
```

Comment:
Are you still calling this `candidateCode` on purpose?

Reply:
Yes, Dominion uses a code which we translate here to the display name. In other files I use `candidateName` not `candidateDisplayName`, but I'm happy to rename the variable here to clarify (or add a comment).

Reply:
Not sure if you missed this comment, but I think it'd be nice to take care of this still.

Reply:
Looks like the comment didn't submit before :) both your comments show as appearing 5 days ago. Yes, I can rename this variable `dominionCandidateId`

Reply:
Oh weird, sorry about that. Looks like we got our wires crossed a bit on this, so I just added a commit to clear it up. Please give it a look over to verify that it makes sense.